src/main/java/dataProvider/ConfigFileReader.java

```diff
@@ -0,0 +1,91 @@
+package dataProvider;
+
+import enums.DriverType;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigFileReader {
+
+    private Properties properties;
+    private final String propertyFilePath = "src/test/resources/configs/configurations.properties";
+
+    public ConfigFileReader() {
+        BufferedReader bufferedReader;
+
+        try {
+            bufferedReader = new BufferedReader(new FileReader(propertyFilePath));
+            properties = new Properties();
+            try {
+                properties.load(bufferedReader);
+                bufferedReader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("configurations.properties not found at " + propertyFilePath);
+        }
+    }
+
+    public String getDriverPath() {
+        String driverPath = properties.getProperty("driverPath");
+        if (driverPath != null) {
+            return driverPath;
+        } else {
+            throw new RuntimeException("driverPath not specified in the configurations.properties file" + propertyFilePath);
+        }
+    }
+
+    public long getImplicitWait() {
+        String implicitWait = properties.getProperty("implicitWait");
+        if (implicitWait != null) {
+            return Long.parseLong(implicitWait);
+        } else {
+            throw new RuntimeException("implicitWait not specified in the configurations.properties file" + propertyFilePath);
+        }
+    }
+
+    public String getWebsiteURL() {
+        String url = properties.getProperty("url");
+        if (url != null) {
+            return url;
+        } else {
+            throw new RuntimeException("url not specified in the configurations.properties file" + propertyFilePath);
+        }
+    }
+
+    public DriverType getBrowser() {
+        String browserName = properties.getProperty("browser");
+        if (browserName == null || browserName.equals("chrome")) {
+            return DriverType.CHROME;
+        } else if (browserName.equalsIgnoreCase("firefox")) {
```

Comment:
in case you have more than 2 **if** blocks, use **switch** statement instead

Reply:
Done